test/parallel/test-readline-interface.js

```diff
@@ -561,15 +561,20 @@ function isWarned(emitter) {
       { input: fi, output: fi, terminal: true }
     );
     const keys = [];
+    const err = new Error('bad thing happened');
     fi.on('keypress', function(key) {
       keys.push(key);
       if (key === 'X') {
-        throw new Error('bad thing happened');
+        throw err;
       }
     });
-    try {
-      fi.emit('data', 'fooX');
-    } catch { }
+    assert.throws(
+      () => fi.emit('data', 'fooX'),
+      (e) => {
```

Comment:
Instead of a validation function, can't this just be `err`?

Reply:
That would also work but I wanted to have a reference equality check. That would not be checked if I'd use `err` directly.